letslambda.json

```diff
@@ -58,7 +145,12 @@
         "LambdaExecutionRole": {
             "Type": "AWS::IAM::Role",
             "Properties": {
-                "ManagedPolicyArns": [ {"Ref": "LetsLambdaManagedPolicy"} ],
+                "ManagedPolicyArns": [
+                    {"Ref": "LetsLambdaManagedPolicy"},
+                    {"Ref": "LetsLambdaS3WriteManagedPolicy"},
+                    {"Ref": "LetsLambdaS3ReadManagedPolicy"},
+                    {"Ref": "LetsLambdaKmsKeyManagedPolicy"}
```

Comment:
This is now documented in `kms-dev`. @petemounce you're also part of the contributors/credits.